homeworks/nagynorbert/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,24 @@
+# 2. homework - Nagy Norbert
+
+#user_inputs
+fict_name = input("Please type your character's name:").strip().capitalize()
+fict_age_days = 365 * int(input("Please add how many years old is your character:"))
+fict_experience = int(input("How many years experencie does your character have in Python programming?"))
+
+#extra
+fict_pro = input("Would your character be a pro Python developer? (Yes/No)").strip().lower()
+
+match fict_pro:
```

Comment:
Szuper a match-case megoldás, de akkor már vigyük tovább :)
Ha nem "yes"-t vagy "no"-t adok meg, akkor a fict_pro_bool változó nem jön létre, és a program egy NameError-ral elhasal ahol később a 19. soron erre a változóra hivatkozol (a case _-nál csak printelsz). Ezt megoldhatnád úgy, hogy addig kérsz be input-ot amíg nem "yes" vagy "no" a válasz (hétfőn tanulunk erről, de tekintve hogy a match/case-t nem tanultuk, lehet már ismered a while loop-okat pl).
Ez persze nem kötelező, csak ha szeretnél egy kis plusz kihívást.